### PR TITLE
Move SkyDNS addon to addons role

### PR DESCRIPTION
**Pull request by altvnk**

### CODE REVIEW

**Comment by altvnk** [comment]
@kenjones-cisco Please review

File: roles/addons/tasks/skydns.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@
   register: dns_rc_def
   when: dns_setup
   tags:
-    - skydns
+    - addons
```
Comment (kenjones-cisco) [inline roles/addons/tasks/skydns.yml:10]: can we keep the skydns tag in addition to addons, that way if someone wants to leverage the tag to specify a specific addon to include?

**Comment by kenjones-cisco** [comment]
Other than the tag comment, LGTM!

File: roles/addons/tasks/skydns.yml (hunk 1 above)
Comment (altvnk) [inline roles/addons/tasks/skydns.yml:10]: Good point, let me change it.

**Comment by altvnk** [comment]
Done, merging

**Pull request merged**